scripts/performance/perf_traffic.py

```diff
@@ -0,0 +1,288 @@
+                                           sample_num: int = 100) -> list:
+        """
+        Generate sample request information.
+
+        :param kind: kind of request.
+        :param sample_num: number of samples will be generated.
+        :return: a list of samples request information.
+        """
+        kinds = ["nym", "schema", "attribute", "claim"]
+
+        if kind not in kinds or sample_num <= 0:
+            return []
+
+        generator = perf_add_requests.PerformanceTesterForAddingRequest(
+            request_num=sample_num, request_kind=kind)
+
+        await generator.test()
+        lst_info = list()
+        with open(generator.info_file_path, "r") as info_file:
+            for line in info_file:
+                if len(line) > 2:
+                    lst_info.append(line)
+
+        try:
+            os.remove(generator.info_file_path)
+        except IOError:
+            pass
+
+        return lst_info
+
+    @staticmethod
+    async def _prepare_samples_for_get_req(sample_num: int = 100):
+        """
+        Init samples for "GET" requests.
+
+        :param sample_num: create a number of samples request information for
+                           each kind of request (nym, attribute, claim, schema)
+        """
+        if TesterSimulateTraffic.__sample_req_info:
+            return
+
+        keys = ["nym", "attribute", "schema", "claim"]
+        if sample_num <= 0:
+            return
+
+        for key in keys:
+            TesterSimulateTraffic.__sample_req_info[key] = \
+                await TesterSimulateTraffic.generate_sample_request_info(
+                key, sample_num)
+
+    @staticmethod
+    def _random_req_kind():
+        """
+        Random choice a request kind.
+
+        :return: request kind.
+        """
+        return random.choice(TesterSimulateTraffic.__kinds_of_request)
+
+    @staticmethod
+    def _random_sample_for_get_request(kind: str):
+        """
+        Choice randomly a sample of request info base on kind of request.
+
+        :param kind: kind of request (get_nym, get_attribute,
+                     get_claim, get_schema).
+        :return: a random sample of request info.
+        """
+        if kind.startswith("get_"):
+            return random.choice(
+                TesterSimulateTraffic.__sample_req_info[kind.replace(
+                    "get_", "")])
+        return ""
+
+    @staticmethod
+    async def _build_and_send_request(sender, args):
+        """
+        Build a request and send it onto ledger.
+
+        :param sender: send the request.
+        :param args: contains some arguments to send request to ledger
+                     (pool handle, wallet handle, submitter did)
+        :return: response time.
+        """
+        kind = TesterSimulateTraffic._random_req_kind()
+        data = TesterSimulateTraffic._random_sample_for_get_request(kind)
+
+        req = await requests_builder.RequestBuilder.build_request(args, kind,
```

Comment:
It looks like we are still waiting for the reply. So, we are waiting for the reply, then sleeping for some random time, and only then sending a new request.
We need to be able to send requests without waiting for results (sending request, say, every 1 sec).